backend/src/types/QueryTypes.ts

```diff
@@ -11,15 +11,6 @@ export interface EventFilterQuery extends PaginationQuery {
     creator?: string;
     location?: string;
     name?: string;
-}
-
-export interface MeetingFilterQuery extends PaginationQuery {
-    before?: string;
-    after?: string;
-    passed?: string;
-    time?: string;
-    creator?: string;
-    location?: string;
-    name?: string;
     roles?: string;
```

Comment:
I think this field might actually not be used, would you be able to remove it please?